ethereum/Webaverse.sol

```diff
@@ -190,7 +188,7 @@ contract Webaverse is WebaverseVoucher, OwnableUpgradeable {
         // make sure signature is valid and get the address of the signer
         address signer = verifyVoucher(voucher);
 
-        _nftContract.mintServerDropNFT(signer, to, name, level, data, voucher);
+        _nftContract.mintServerDropNFT(signer, to, name, data, voucher);
```

Comment:
Not sure the name is needed if it's in the IPFS.

Reply:
Removed